### ISSUE TITLE
wqPane.addItem is deprecated.

### ISSUE THREAD

**Issue by benphelps** [Fred-Barclay/Termination#26]
Pane::addItem(item, 1) is deprecated in favor of Pane::addItem(item, {index: 1})
```
Pane.addItem (/Applications/Atom Beta.app/Contents/Resources/app.asar/src/pane.js:462:14)
StatusBar.onDropTabBar (/Users/benphelps/.atom/packages/termination/lib/status-bar.coffee:717:12)
onDropTabBar (/Users/benphelps/.atom/packages/termination/lib/status-bar.coffee:3:59)
HTMLUListElement.<anonymous> (/Users/benphelps/.atom/packages/termination/lib/status-bar.coffee:366:26)
HTMLUListElement.dispatch (/Users/benphelps/.atom/packages/termination/node_modules/jquery/dist/jquery.js:4435:9)
HTMLUListElement.elemData.handle (/Users/benphelps/.atom/packages/termination/node_modules/jquery/dist/jquery.js:4121:28)
```

**Comment by benphelps** [comment]
Reproducible by dragging a terminal tab to the files tab bar, which mostly works.

**Comment by Fred-Barclay** [comment]
Thanks! Merged and pushed to the repo. We'll ship this in the next release. :smile:

**Comment by lordofdestiny** [comment]
I just installed Atom and Termination and I'm getting this deprecation now. Is it really fixed, because my Atom freaked out?

**Comment by Fred-Barclay** [comment]
@lordofdestiny yep, it was accidentally reverted in d4190a1c2773bfc2bd517178a1fc138a4c85e4e3. :fearful: Thanks for reporting! Fixed in 4e8ed893b81960894cdfe10692181906f1b50372

That said, dragging a terminal tab to the file tab bar no longer works and I'm not quite sure why...
